lib/node_modules/@stdlib/math/base/special/README.md

```diff
@@ -266,7 +266,7 @@ var fcns = special;
 -   <span class="signature">[`erf( x )`][@stdlib/math/base/special/erf]</span><span class="delimiter">: </span><span class="description">error function.</span>
 -   <span class="signature">[`erfc( x )`][@stdlib/math/base/special/erfc]</span><span class="delimiter">: </span><span class="description">complementary error function.</span>
 -   <span class="signature">[`erfcinv( x )`][@stdlib/math/base/special/erfcinv]</span><span class="delimiter">: </span><span class="description">inverse complementary error function.</span>
--   <span class="signature">[`erfcx( x )`][@stdlib/math/base/special/erfcx]</span><span class="delimiter">: </span><span class="description">scaled complementary error function.</span>
```

Comment:
@Planeshifter Not clear why the argument was removed.

Reply:
Most recent snapshot: https://github.com/stdlib-js/stdlib/tree/4b1d77d2bd001d5970ce93230765a579fb41349d/lib/node_modules/%40stdlib/math/base/special/erfcx

Reply:
> We have a heuristic that checks the README.md files to see whether there are multiple function signatures, in which case we omit them from the generated table of contents.

@Planeshifter We can probably improve that heuristic by checking whether the listed signatures actually differ. If they are equal, no need to assume variadic.